### DISCUSSION TITLE
No color set for remote window buttons?

### DISCUSSION THREAD

**Discussion by cateiru** [primer/github-vscode-theme#301] [Q&A] [answered]
![スクリーンショット 2022-07-15 10 51 05](https://user-images.githubusercontent.com/24271196/179130811-aacdc639-949e-47c3-a7e5-716ce6d7dca5.png)

The remote window buttons appear to have no color set.
I am wondering if this is a github-vscode-theme issue, as the default VSCode theme has the colors set correctly.

Has anyone else reproduced this problem? Or is this a spec?

Default theme:
![スクリーンショット 2022-07-15 10 53 22](https://user-images.githubusercontent.com/24271196/179131055-baf2e377-fbcf-453f-8228-324a88d933f8.png)

environments

- VSCode
  - version: 1.69.1 (Universal)
  - commit: b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a
  - date: 2022-07-12T08:21:51.333Z
  - Electron: 18.3.5
  - Chromium: 100.0.4896.160
  - Node.js: 16.13.2
  - V8: 10.0.139.17-electron.0
  - OS: Darwin x64 21.5.0
- github-vscode-theme
  - v6.2.0

**Comment by simurai** [comment, marked as answer]
Yes, this was changed intentionally with https://github.com/primer/github-vscode-theme/pull/172.

I guess the idea was that since the rest of the status bar doesn't have a background color (like the blue in the default theme), the remote button should also not have a specific background color to match the rest.

@cateiru do you miss having the remote button stand out more?

**Reply by cateiru** [reply]
Thanks for the reply.

I often work remotely using SSH and often check to see if I am connected, so it was easier to see if the remote buttons were colored.

(I hope you consider my opinion as a reference only)
(I like the new design, but I also like the previous colored version.)

**Reply by simurai** [reply]
It would be nice if there is a way to color the button background only when the remote connection is **active**. Then it's nicely subtle by default, but highlighted when connected. But I think that's not possible and it's just an always or never. 🤔
